WooCommerce/Classes/ViewRelated/CardPresentPayments/CardReaderConnectionController.swift

```diff
@@ -103,13 +103,18 @@ final class CardReaderConnectionController {
 
     private var onCompletion: ((Result<Bool, Error>) -> Void)?
 
+    /// Gateway ID to include in tracks events
+    private var gatewayID: String?
```

Comment:
This can be a `let`, I think

Reply:
With the changes in https://github.com/woocommerce/woocommerce-ios/pull/6123/commits/06741436c8b245fc5c691e3537ebc86f0e974d11 this needs to remain a var